mobile/library/cc/engine_builder.cc

```diff
@@ -491,6 +491,10 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
 #else
   if (use_cares_) {
     envoy::extensions::network::dns_resolver::cares::v3::CaresDnsResolverConfig resolver_config;
+    auto* address = resolver_config.add_resolvers();
+    address->mutable_socket_address()->set_address("8.8.8.8");
```

Comment:
I think we should also consider the IPv6 DNS addresses: https://developers.google.com/speed/public-dns/docs/using

If we are on an IPv6-only network, I don't know if adding 8.8.8.8 is going to cause any issues. We should definitely verify this first.

Reply:
I tried configuring only `8.8.8.8` on a IPv6 only network and it worked fine. My guess is the ipv4 gets mapped to an ipv6 address somewhere.
I agree adding a ipv6 resolver feels safer though. done.

Reply:
I think having a knob is better than hardcoding it given that other users may want to use different DNS servers, such as Cloudflare, etc.